cpanfile

```diff
@@ -5,7 +5,7 @@ requires 'perl', '5.26.0';
 
 # basics
 requires 'Data::UUID';
-requires 'Data::Printer';
+requires 'Data::Printer', '0.99_019', dist => 'GARU/Data-Printer-0.99_019.tar.gz';
```

Comment:
Why a dep on an unreleased version? Are we using DDP for actual business logic?

Reply:
As described in the commit message, to get off of File::HomeDir and Mac::SystemDirectory.

We only use Data::Printer in development debugging.  It is `use`d in a few `lib` files, but not actually called. All references to it can likely be removed safely, but I suspect some contributors might still want it to be installed. (Personally I use Data::Dumper during development, so I'm ambivalent.)

Reply:
I'd personally remove it from the deps and then let people install it locally as they see fit.